starter-code/bin/seed.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require('mongoose');
+const User = require('../models/user');
+mongoose.connect('mongodb://localhost/lab-basic-auth', {useMongoClient: true});
+
+const user = new User({
+  username: 'Thibaut',
+  password: '1234'
```

Comment:
Now that your passwords are hashed it may be a good idea to do the same for the seed.